pyiron_contrib/RDM/storagejob.py

```diff
@@ -0,0 +1,337 @@
+import os
+import posixpath
+import shutil
+
+from pyiron_base import GenericJob, DataContainer
+from pyiron_base.generic.filedata import FileData
+from pyiron_contrib.generic.s3io import FileS3IO
+
+
+def _remove_s3_working_directory(file_s3_io_handle, group_to_remove):
```

Comment:
Does it make sense have this live on `file_s3_io_handle`, so that `_before_generic_remove_child` below can just call the method?

Reply:
I moved it into `_before_generic_remove_child`